src/domain/card/CardPasswordNumber/CardPasswordNumber.stories.tsx

```
@@ -0,0 +1,67 @@
+
+    return (
+      <CardPasswordNumber
+        cardPassword={cardPassword}
+        cardPasswordErrorMessage={cardPasswordErrorMessage}
+        handleCardPasswordInputChange={handleCardPasswordInputChange}
+      />
+    );
+  },
+};
+
+export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)